### PR TITLE
Fix hang when incoming p2p or group calls throw up dialog.

### PR DESCRIPTION
**Pull request by roxanneskelly**
Fix for https://github.com/secondlife/viewer/issues/2346

There were changes in atlasaurus that resulted in a hang for incoming p2p and group calls which throw up dialogs.  The changes revolved around mutex, coroutines, job queues, and such.
The fix was to do any processing that may result in callbacks from the webrtc code in a queued job instead of a coroutine.

### CODE REVIEW

**Review by akleshchev — APPROVED** [review]

File: indra/newview/llvoicewebrtc.cpp [hunk 1]
```diff
@@ -549,13 +549,20 @@ void LLWebRTCVoiceClient::voiceConnectionCoro()
...
-                updateOwnVolume();
-            }
+            LL::WorkQueue::postMaybe(mMainQueue,
+                [=] {
+                    if  (sShuttingDown)
+                    {
+                        return;
+                    }
+                    sessionState::processSessionStates();
+                    //sessionState::processSessionStates();
```
Comment (brad-linden) [inline indra/newview/llvoicewebrtc.cpp:559]: commented out code got left in?

Comment (roxanneskelly) [reply indra/newview/llvoicewebrtc.cpp]: oopse :)

**Comment by roxanneskelly** [comment]
As noted in the code, this fix is webrtc only, so set the priority for merging accordingly.

**Comment by roxanneskelly** [comment]
Likely cause of https://github.com/secondlife/viewer/issues/2376 as well.

**Review by brad-linden — APPROVED** [review]

**Pull request merged**